src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/AuthWebClientConfiguration.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.config
+
```

Comment:
This file is just the old WebClientConfig file (not sure why it doesn't show as rename)